docs-developer/release_process.rst

```diff
@@ -85,36 +84,33 @@ Ensure bugfixes from internal depencies have propagated
 Some issues in Kolibri arise due to our integration of internally produced, but external to Kolibri, packages, such as kolibri-exercise-perseus-renderer, iceqube, and morango. If any of these kinds of dependencies have been updated to fix issues for this milestone, then the dependency version should have been updated.
 
 
-Edit the VERSION file
-~~~~~~~~~~~~~~~~~~~~~
+Bump the version
+~~~~~~~~~~~~~~~~
 
-Current practice is to bump ``kolibri.VERSION`` before tagging a release. You are allowed to have a newer version in ``kolibri.VERSION``, but you are not allowed to add the tag before actually bumping ``kolibri.VERSION``.
+.. note:: This applies to all releases, both stable and pre-releases. Anything that's released should be tracked in a release branch.
```

Comment:
Note: this was not what either @indirectlylit or myself had understood about the design of the versioning system. The PR I opened yesterday was to try to bring the versioning system into line with what we had expected.

Reply:
I keep re-learning this for some reason :smile: 

But it's fine, let's leave it.. I can't see making non-tracked pre-releases going all that wrong, after all the tag is there.